src/topics/recent.js

```diff
@@ -63,6 +64,10 @@ module.exports = function(Topics) {
 
 	Topics.updateRecent = function(tid, timestamp, callback) {
 		callback = callback || function() {};
-		db.sortedSetAdd('topics:recent', timestamp, tid, callback);
+		if (plugins.hasListeners('filter:topics.updateRecent')) {
+			plugins.fireHook('filter:topics.updateRecent', {tid: tid, timestamp: timestamp}, callback);
```

Comment:
or this might be better

if (plugins.hasListeners('filter:topics.updateRecent')) {
    plugins.fireHook('filter:topics.updateRecent', {tid: tid, timestamp: timestamp}, function(err) {
        if (err) db.sortedSetAdd('topics:recent', timestamp, tid);
        callback();
    });
}